geoh5py/objects/cell_object.py

```diff
@@ -53,6 +52,7 @@ def cells(self) -> np.ndarray:
         """
         Array of indices defining connecting vertices.
         """
+        # todo: technically this is not possible anymore
```

Comment:
It is possible during a copy with `clear_cache` options. It was a feature added to avoid increasing the memory requirements on the copy of large files.